tests/login.test.js

```diff
@@ -24,4 +24,27 @@ describe("login endpoint test", () =>{
       expect(res.status).toEqual(401);
     });
   });
+
+  describe("email is not in body", () =>{
+    it("should return 400", async() =>{
+      const res = await request(app)
+        .post("/login")
+        .set("Accept", "application/json")
+        .send({
+          password: "Password"
+        });
+      expect(res.status).toEqual(400);
+    });
+  });
+  describe("password is not in body", () =>{
+    it("should return 400", async() =>{
+      const res = await request(app)
+        .post("/login")
+        .set("Accept", "application/json")
+        .send({
+          email: "txiong@"
+        });
+      expect(res.status).toEqual(400);
+    });
+  });
```

Comment:
Add a test to see that we are getting a cookie called `user`.

Reply:
https://github.com/The-Silent-Corner/sad-twt/pull/47/commits/e2aa7ae74563d57df3e3f1270415b62c74ec7d77?file-filters%5B%5D=.js#diff-b336af6a9e096ca62b60db8885a87fdc0b112d9d755520eedbf58b75af02795eR50

Reply:
see https://github.com/The-Silent-Corner/sad-twt/commit/e2aa7ae74563d57df3e3f1270415b62c74ec7d77?file-filters%5B%5D=.js#r50439450

Reply:
ed1603e32fd81fea73504fc6bb0585c50b05580c